examples/HH_simulator.ipynb

```diff
@@ -628,7 +628,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "infer = SnpeC(sample_with_mcmc=False, num_pilot_sims=2000, **snpe_common_args)\n",
+    "infer = SnpeC(sample_with_mcmc=False, **snpe_common_args)\n",
```

Comment:
I guess this might be changed by @michaeldeistler PR on HH?

Reply:
You mean that it might be reverted? On rebasing he will probably find this as a conflict @michaeldeistler